lib/backend/watchersyncer/watchersyncer.go

```diff
@@ -62,27 +64,36 @@ type SyncerUpdateProcessor interface {
 
 // New creates a new multiple Watcher-backed api.Syncer.
 func New(client api.Client, resourceTypes []ResourceType, callbacks api.SyncerCallbacks) api.Syncer {
+	backoffUpdates := make(chan time.Time)
+	// Create a backoff handler that will increase the backoff time by 100 ms every time a backoff
+	// is required until a maximum of 5 seconds is reached. The backoff time will slowly decrease by
+	// 10 ms every second until the minimum backoff time of 100 ms is reached.
+	bh := backoff.NewLinearBackoffHandler(backoffUpdates, 10*time.Millisecond, 1*time.Second, 100*time.Millisecond, 5*time.Second, 100*time.Second)
```

Comment:
Is this right? Do you mean to increment the backoff by 100 seconds instead of 100 milliseconds?